proj4/meta.yaml

```diff
@@ -7,10 +7,11 @@ source:
   url: http://download.osgeo.org/proj/proj-4.9.1.tar.gz
 
 build:
+  number: 0  # [osx]
   number: 1  # [not osx]
 
-requirements:
-  run:
+requirements:  # [win]
+  run:  # [win]
```

Comment:
Yeah, afraid so. Without them we end up with:

```
requirements:
   run:

about:
   home: ...
```

And conda barfs as it thinks we've passed an empty string to the run requirement. Opportunity for a conda-build PR if you're interested?